apps/judicial-system/web/messages/RestrictionCases/Prosecutor/requestedHearingArrangementsForm.ts

```diff
@@ -9,6 +9,23 @@ export const rcRequestedHearingArrangements = {
       'Notaður sem titill á óskir um fyrirtöku skrefi í gæsluvarðhalds- og farbannsmálum.',
   }),
   sections: {
+    prosecutor: defineMessages({
+      heightenSecurityLevelLabel: {
+        id:
+          'judicial.system.investigation_cases:requested_hearing_arrangements.prosecutor.heighten_security_level_label',
```

Comment:
Here, the id should be "...restriction_cases...".

The pull request states the heightened security is being added to investigation cases, but the in reality it is being added to restriction cases.

The same comment applies in several places below.